client/nui/index.html

```diff
@@ -76,7 +78,7 @@
 				<div class="status-bar">
 					<div class="battery-life">
 						<i class="fa-solid fa-battery-three-quarters icon"></i>
-						<span>69%</span>
+						<span>69%</span>&nbsp;&nbsp;<b id="flashstatus" class="off"><i class="fa-solid fa-bolt"></i></b>
```

Comment:
Instead of adding the `&nbsp;` twice, just add a `margin-left` to the `flashstatus` selector in the CSS.